modules/perception/inference/migraphx/plugins/plugin.h

```diff
@@ -0,0 +1,128 @@
+/******************************************************************************
```

Comment:
@B1tway, I believe, we've discussed it.

Reply:
Yes, this file is based on TensorRT code, so we need to add the proper license

Reply:
I think the best way is to get rid of this part. In the final version of MIGraphX inference this part is not necessary and after refactoring it can be removed

Reply:
@B1tway I see that `plugin.h` is a dependency for other plugins; how to remove it safely? If we want to add a license, then Apollo should include "This software contains source code provided by NVIDIA Corporation.” in its license as per https://docs.nvidia.com/deeplearning/tensorrt/sla/index.html which may not be acceptable. Can we remove `plugin.h` and refactor it today?

Reply:
@B1tway Can you elaborate on which part to remove? the Dims class is being used which comes from plugin.h so we can't directly remove it.

Reply:
Done
https://github.com/amd/apollo/commit/fa01c4039d81487991a440dfc11c145fa272f1f4